test/unit/lib/plugins/aws/package/compile/events/stream.test.js

```diff
@@ -316,388 +319,59 @@ describe('AwsCompileStreamEvents', () => {
     });
 
     describe('when a DynamoDB stream ARN is given', () => {
-      it('should create event source mappings when a DynamoDB stream ARN is given', () => {
+      it('fails if Ref/dynamic stream ARN is used without defining it to the CF parameters', () => {
```

Comment:
Let's not edit any old tests. They should be removed (when replaced), or kept intact

Reply:
@olgashi in case where old test was just partially replaced, let's simply keep it untouched